samples/application/simple-cli-go/Makefile

```diff
@@ -16,5 +16,6 @@ DOCKER_BUILD_OPTS += --no-cache
 docker:
 	$(DOCKER) build $(DOCKER_BUILD_OPTS) \
     	-t fpc/fpcclient:$(FPC_VERSION) \
+		-t fpc/fpcclient:latest \
     	-f Dockerfile \
```

Comment:
```suggestion
    	-t fpc/fpcclient:$(FPC_VERSION) \
	-t fpc/fpcclient:latest \
    	-f Dockerfile \
```

Reply:
I don't know why you see a larger indentation ... in my vscode they all are perfectly aligned.